screenpy/actions/eventually.py

```diff
@@ -138,15 +138,15 @@ def perform_as(self, the_actor: Actor) -> None:
                     return
                 except Exception as exc:  # pylint: disable=broad-except
                     self.caught_error = exc
-                    self.unique_errors.add((exc.__class__.__name__, str(exc)))
+                    self.unique_errors[exc] = None
```

Comment:
I'm so happy this is the solution you went with. This is such a funny use of `dict`.